examples/ct/handlers.go

```diff
@@ -337,20 +338,20 @@ func getSTH(c LogContext, w http.ResponseWriter, r *http.Request) (int, error) {
 	}
 
 	// Now build the final result object that will be marshalled to JSON
-	jsonResponse := getSTHResponse{
+	jsonRsp := getSTHResponse{
```

Comment:
How about you let me use `rsp` in `examples/ct/handlers.go` and I let you use `resp` in `server/log_rpc_server.go` ? :-)